versions.yaml

```diff
@@ -117,7 +117,7 @@ assets:
     architecture:
       aarch64:
         name: &default-image-name "ubuntu"
-        version: &default-image-version "latest"
+        version: &default-image-version "jammy"
```

Comment:
I think pinning the version rather than using `latest` is a good idea. I thought we were using `20.04` anyway, so do you understand why `latest` was being ignored?

Reply:
Take a look at the very first commig.
We were passing `IMG_OS_VERSION` to the script, which was not being taken into consideration at all.

It'd lead to using 20.04 dockerfiles, which would enforce 20.04 was used as rootfs.

Reply:
btw, another thing, the fact our scripts to build this from the kata-deploy/local-build uses 20.04 is what ended up enforcing the dockerfile to be generated based on 20.04, due to the env var propagation not being correctly done. :-)

Reply:
Is this desirable behaviour to default to the builder image, or is https://github.com/kata-containers/kata-containers/blob/b4c710576e1433d596c657f4fb5799b1dc69f23c/tools/osbuilder/rootfs-builder/ubuntu/config.sh#L8
the place to put [ -z "$OS_VERSION" ] && die "needs an OS VERSION" (or a `set -u` to do it automatically)?

Reply:
I think the best we can do is to remove the UBUNTU_CODENAME option, and then die if `OS_VERSION` is not set.

Reply:
Yeah, I think that would help - just trying to find a way to detect a bug that we've now found in the unlikely event it appears again.